manylinux1-wheel-build/README.md

```diff
@@ -0,0 +1,31 @@
+# Wheel Builder image
+
+This image is a little different than the other images in this repo --
+it's intended to be a quick way to build manylinux wheels with the
+production version of the libraries for debugging and testing
+purposes. This is designed to work as a single docker image that
+doesn't require privlege or multiple images to run.
+
+Like the other images in this repo, it expects that there is a Pillow
+source directory mounted at /Pillow. Unlike the others, it puts the
+output files in /output. This should be mounted as a volume to the host
```

Comment:
```suggestion
output files in /out. This should be mounted as a volume to the host
```
Everything else in these changes suggests it is 'out', not 'output'?

Reply:
```
docker run --rm -v $(ROOT):/Pillow -v `pwd`/out:/output $(IMAGENAME):$(BRANCH) /build.sh $(_PYVER)
```
Maps ./out to /output in the container. 

```
auditwheel repair /tmp/Pillow*whl -w /output
```
Puts the wheel in the /output directory in the container.